jekyll/_posts/guides/2018-09-21-usage-of-the-matrix-js-sdk.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: post
+categories: guides
+---
+
+<style>
+ul {
+    list-style: circle;
+    margin-left: 30px;
+}
+p {
+    padding-bottom: 1em;
+    line-height: 1.7em;
+}
+h2 {
+    margin-bottom: 15px;
+    margin-top: 30px;
+}
+
+</style>
+
+# Usage of the matrix-js-sdk
+
+Matrix allows real-time communications over the Internet using HTTP and JSON. This makes developing clients to connect to Matrix servers really easy! Because it's open, and uses simple syntax for messages, you can connect Matrix to anything that communicates over a standard HTTP interface - later projects in this series will explore ideas such as building bots, performing machine learning on message content, and connecting IoT devices such as Philips Hue lights.
+
+## Making a Matrix Client
+
+Let's explore how we would make a very simple Matrix client, with only the ability to perform an initial sync, and to get member lists and the timeline for rooms of our choice.
+
+This article will explore the [Matrix Client-Server API], making use of the [matrix-js-sdk]. Later articles may discuss making the underlying calls. Specifically we will cover:
+
+* login
+* simple syncing
+* listening for timeline events
+* access the `MatrixInMemoryStore`
+* sending messages to rooms
+
+We'll use Node.js as our environment, though the [matrix-js-sdk] can also be used directly in the browser.
+
+## Let's write JavaScript
+
+The first thing to do is install the [matrix-js-sdk] from NPM:
```

Comment:
we should prolly include a `mkdir myproject; cd myproject` or something in here for real noobs, and a link through to https://nodejs.org so folks can grab node & npm?